src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -59,4 +63,14 @@ public void deleteLineById(Long id) {
 		Line line = getById(id);
 		lineRepository.delete(line);
 	}
+
+	public LineResponse updateSections(Long id, SectionRequest sectionRequest) {
+		Section newSection = sectionRequest.toSection();
+		Line line = getById(id);
```

Comment:
라인에 새로운 구간을 추가할때는 추가되는 구간 객체를 생성할 때 line을 설정해줄 수도 있을 것 같아요 :)
```suggestion
                 Line line = getById(id);
		Section newSection = sectionRequest.toSection(line);
```

Reply:
아 line 이 빠졌네요 감사합니다!